app/templates/components/queued-jobs.hbs

```diff
@@ -1,37 +1,41 @@
 {{#if this.jobs.isLoaded}}
-  {{#each this.jobs as |job|}}
-    <div
-      data-test-sidebar-queued-job
-      class="tile tile--sidebar {{job.state}}"
-    >
-      {{#if job.repo.slug}}
-        <StatusIcon @status={{job.state}} />
-        <LinkTo @route="job" @models={{array job.repo job}}>
-          {{job.repo.slug}}
-        </LinkTo>
-      {{/if}}
-      <p class="tile-title float-right">
-        <SvgImage @name="icon-hash" @class="icon" />
+  {{#if this.queuedJobs.length}}
```

Comment:
This `{{#if}}{{/if}}` in not necessary as `{{#each}} list {{else}} list empty {{/each}}` can handle it.